harfbuzz-sys/build.rs

```diff
@@ -11,14 +12,24 @@ fn main() {
         }
     }
 
-    assert!(Command::new("make")
-        .args(&["-R", "-f", "makefile.cargo", &format!("-j{}", env::var("NUM_JOBS").unwrap())])
-        .status()
-        .unwrap()
-        .success());
+    // On Windows, HarfBuzz configures atomics directly; otherwise,
+    // it needs assistance from configure to do so.  Just use the makefile
+    // build for now elsewhere.
+    let target = env::var("TARGET").unwrap();
+    if target.contains("windows") {
+        let dst = cmake::Config::new("harfbuzz").build();
+        println!("cargo:rustc-link-search=native={}", dst.display());
```

Comment:
Because of the `install(TARGETS harfbuzz ARCHIVE DESTINATION .)`.  I can make this go to lib for consistency if you'd prefer.